seqr/utils/elasticsearch/es_search.py

```diff
@@ -990,11 +992,13 @@ def _get_paginated_searches(self, index_name, page=1, num_results=100, start_ind
                     # ES request size limits are limited by offset + size, which is the same as end_index
                     from seqr.utils.elasticsearch.utils import InvalidSearchException
                     raise InvalidSearchException(
-                        'Unable to load more than {} variants ({} requested)'.format(MAX_VARIANTS, end_index))
+                        'Unable to load more than {} variants ({} requested)'.format(MAX_VARIANTS, end_index),
+                    )
```

Comment:
Are there any changes?

Reply:
moved the end parenthesis to the next line for some reason. I can switch it back